libp2p-playground/relay/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# ---------- builder ---------
```

Comment:
Maybe we can use only one Dockerfile. The first one is a builder, the second one is a "base" with dependencies (iproute2-tc netcat-openbsd, ...) and then we can put separate binaries into "base" image.